### DISCUSSION TITLE
Shape function support

### DISCUSSION THREAD

**Discussion by mnuriyumusak** [timeplus-io/proton#774] [Q&A]
Is this tool has shape operation support like linestring, polygon, points etc. 
my need is this, i have textual linestring data type, i want to find if this linestring is in polygon or not. this is not geographical but in cartesian coordinate system. is proton has such function, if not how to achieve this? thanks.

**Comment by jovezhong** [comment]
Hi @mnuriyumusak , would you like to check https://docs.timeplus.com/functions_for_geo#point_in_polygon and we certainly can add more such functions if you can share a bit more use cases. I will convert this issue to a discussion.
